tests/rptest/tests/while_busy_test.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2022 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+# December 2021, engineers were tasked to identify failures with one of Redpanda's latest features, shadow indexing.
+# Many new tests and workloads were created as a result of this effort. These tests, however, were setup and run manually.
+# Within this file are those same tests but written and run as ducktape tests.
+
+from ducktape.mark.resource import cluster
+from ducktape.utils.util import wait_until
+from ducktape.cluster.cluster_spec import ClusterSpec
+
+import os
+import time
+import requests
+import random
+from math import ceil
+
+from rptest.clients.rpk import RpkTool
+from rptest.clients.types import TopicSpec
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.redpanda import SISettings
+from rptest.services.franz_go_verifiable_services import FranzGoVerifiableProducer, FranzGoVerifiableSeqConsumer, FranzGoVerifiableRandomConsumer
+from rptest.services.franz_go_verifiable_services import ServiceStatus
+
+
+class WhileBusyTest(RedpandaTest):
```

Comment:
Any name suggestions? I'm not sure "WhileBusyTest" is the most descriptive name.